Automattic/wp-calypso
# SMP: Add connect jetpack into the split button

#### Proposed Changes

* Refactor Add New Site to be a Split button
* Add "Connect Jetpack Site" as a popover menu item inside the split button.
* Send `cta_from` query param to be tracked in the event `calypso_jpc_url_view`
* SplitButton: accept `href` new prop
* SplitButton accept `whiteSeparator` new prop

#### Screenshots

| Before | After |
|---|---|
| <img width="267" alt="67646-before" src="https://user-images.githubusercontent.com/779993/189676529-7b5ef304-14ff-4067-866e-bc4fb6b903af.png">   |  <img width="273" alt="Screenshot 2022-09-13 at 15 21 22" src="https://user-images.githubusercontent.com/779993/189926471-e66ca00a-3afd-43ec-8f27-7832888dba16.png"> |


#### Testing Instructions



* Go to `/sites`
* Observe the `Add new site` on the top-right corner is a split button
* Click on it
* Observe you go to the add new site flow as you would do before this PR.
* Click on `< Back to Sites` to go back to `/sites`
* On the top-right corner, click on the small arrow of the split button.
* Observe that a menu appears
* Click on `Connect Jetpack site`
* Observe that you are redirected to `/jetpack/connect`

#### Pre-merge Checklist






- Closes #67646
